index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <title> About Jei</title>
+</head>
+<body>
+    <header>
+        <hgroup></hgroup>
+        <h2>ABOUT JEI:</h2>
+    </header>
+
+    <img src="img/jei.jpg" alt="island cocktail"/>
+    <div id="words">
+        <p>Jei has over 20 years of experience as a teacher and social worker.
```

Comment:
```suggestion
        <p>Jei has over 20 years of experience as a teacher and social worker.</p>
```
Missing closing paragraph tag.